bedrock/wordpress/models.py

```diff
@@ -92,16 +107,11 @@ def get_absolute_url(self):
         return self.link
 
     def htmlify(self):
-        summary = strip_tags(self.excerpt).strip()
-        if summary.lower().endswith('continue reading'):
-            summary = summary[:-16]
-
-        return Markup(summary)
+        return Markup(self.excerpt)
```

Comment:
Note that [according to the docs](https://docs.djangoproject.com/en/1.11/ref/utils/#django.utils.html.strip_tags) `strip_tags` is not to be considered safe.